pandas/core/indexes/base.py

```diff
@@ -4428,9 +4438,12 @@ def argsort(self, *args, **kwargs):
         >>> idx[order]
         Index(['a', 'b', 'c', 'd'], dtype='object')
         """
+
```

Comment:
Not a huge deal so no action required, but just a general comment  for large-is PRs better to leave things like this alone to minimize diff

Reply:
I reverted this in the latest commit.